src/dotnet/commands/dotnet-add/dotnet-add-p2p/Program.cs

```diff
@@ -81,5 +118,10 @@ public static int Run(string[] args)
                 return 1;
             }
         }
+
+        private static string GetFrameworkDisplayString(NuGetFramework framework)
```

Comment:
I'm removing it since I'll use it in one place only after doing some slight changes